### PR TITLE
Cascade restriction attribute

### PR DESCRIPTION
**Pull request by CBroz1**
- Adds line to solve #1159 
- Adds test of edge case demonstrated in above issue
- Tests for cascading delete pass locally
- Pytests show unrelated errors related to an SSL certificate

> WARNING  urllib3.connectionpool:connectionpool.py:874 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1002)'))': /datajoint.test

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - Added - Missing tests for set_password - PR [#1106](https://github.com/datajoint/datajoint-python/pull/1106)
 - Changed - Returning success count after the .populate() call - PR [#1050](https://github.com/datajoint/datajoint-python/pull/1050)
 - Fixed - `Autopopulate.populate` excludes `reserved` jobs in addition to `ignore` and `error` jobs 
+- Fixed - `cascade` passes `_restriction_attributes` when passing `_restriction` - PR [#1160](https://github.com/datajoint/datajoint-python/pull/1060)
```
Comment (dimitri-yatsenko) [inline CHANGELOG.md:14]:
```suggestion
- Fixed - Issue [#1159]((https://github.com/datajoint/datajoint-python/pull/1059)  (cascading delete)  - PR [#1160](https://github.com/datajoint/datajoint-python/pull/1060)

**Review by dimitri-yatsenko — CHANGES_REQUESTED** [review]
This is a really good catch.  Thank you.

**Review by dimitri-yatsenko — APPROVED** [review]

**Comment by ethho** [comment]
> Pytests show unrelated errors related to an SSL certificate

`docker pull datajoint/nginx:latest` before running tests should fix this. CI always pulls latest image, which is why this doesn't appear in GHA runs.
